ds-css/src/styles/main.scss

```diff
@@ -1,4 +1,95 @@
 @charset "utf-8";
+
+// INTRODUCTION: for customizations to matching CBP general variables.
+
+// Everything is in alphabetical order w/in each section
+
+// ** Only general variables, nothing pertaining to a specific element/component/object
+// Note: For all customizations to non-generic variables, need to find inside those more specific files.
+
+// Beginning of General Variable Customizations
+
+// Set CBP initial colors
+
+// Overidden variables 
+// (* w/in the /initial-variables.sass Bulma file)
+$grey:                              #555555;
+$grey-dark:                         #333333;
+$grey-darker:                       #222222;
+$grey-light:                        #aaaaaa;
+$grey-lighter:                      #eeeeee;
+$white:                             #ffffff;
+
+// New variables to Bulma
```

Comment:
What is our standard for hex? Caps or lower case?

Reply:
ive always gone by the universal standard of keeping everything lower case

Reply:
Great, let's make everything lower case then.

Reply:
ok, gotcha